src/main/java/brandkon/Brand/BrandService.java

```diff
@@ -0,0 +1,35 @@
+package brandkon.Brand;
+
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class BrandService {
+
+    private final BrandRepository brandRepository;
+
+    public BrandService(BrandRepository brandRepository) {
+        this.brandRepository = brandRepository;
+    }
+
+    public List<BrandResponse> listAllBrand(){
+        return brandRepository.findAll()
+                .stream()
+                .map(brand -> new BrandResponse(brand.getId(),
+                        brand.getName(),
+                        brand.getImageUrl())).toList();
```

Comment:
```suggestion
                .map(brand -> new BrandResponse(brand.getId(),
                        brand.getName(),
                        brand.getImageUrl()))
                .toList();
```
줄바꿈을 해서 가독성을 좀 더 높일 수 있답니다